infra/modules/grafana-agent/main.tf

```diff
@@ -55,3 +55,64 @@ resource "helm_release" "grafana_agent" {
   VALUES
   ]
 }
+
+resource "helm_release" "grafana-k8s-monitoring" {
+  name       = "grafana-k8s-monitoring"
+  repository = "https://grafana.github.io/helm-charts"
+  chart      = "k8s-monitoring"
+  version    = "0.1.4"
+  namespace  = kubernetes_namespace.grafana-agent.metadata[0].name
+
+  values = [ <<-VALUES
+    cluster:
+      name: dapps-prod-us-east-1
+    externalServices:
+      prometheus:
+        host: https://prometheus-us-central1.grafana.net
+        basicAuth:
+          username: ${var.grafana-prom-username}
+          password: ${data.kubernetes_secret.api_key_secret.data["GRAFANA_PASSWORD"]}
+      loki:
+        host: https://logs-prod-017.grafana.net
+        basicAuth:
+          username: ${var.grafana-loki-username}
+          password: ${data.kubernetes_secret.api_key_secret.data["GRAFANA_PASSWORD"]}
+    metrics:
+      cost:
+        enabled: false
+      podMonitors:
+        namespaces:
+          - "marlowe-staging"
+          - "dapps-certification-staging"
+      probes:
+        namespaces:
+          - "marlowe-staging"
+          - "dapps-certification-staging"
+      serviceMonitors:
+        namespaces:
+          - "marlowe-staging"
+          - "dapps-certification-staging"
+    opencost:
```

Comment:
What about the other namespaces?

dapps-certification
marlowe-production

Reply:
Those two did not contain deployments as at the time of this PR, but I see marlowe-production now has some pods running in it. I'll update now.